modules/tinymce/src/themes/silver/test/ts/browser/editor/toolbar/SplitFloatingToolbarKeyboardNavigationTest.ts

```diff
@@ -0,0 +1,43 @@
+import { FocusTools, Keys } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyHooks, TinyUiActions } from '@ephox/mcagar';
+import { Attribute, SugarDocument } from '@ephox/sugar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.themes.silver.editor.toolbar.SplitFloatingToolbarKeyboardNavigationTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    menubar: false,
+    width: 1000,
+    setup: (ed: Editor) => {
+      ed.ui.registry.addMenuButton('menubutton', {
+        icon: 'bookmark',
+        tooltip: 'menubutton',
+        fetch: (success) => {
+          success('bold strikethrough');
+        }
+      });
+    },
+    toolbar: 'redo undo | styles | bold italic underline strikethrough forecolor backcolor | removeformat alignleft aligncenter alignright alignjustify | indent outdent |'
+  + ' fontfamily fontsize | bold italic | lignheight | print underline menubutton alignright | alignleft aligncenter',
+    base_url: '/project/tinymce/js/tinymce'
+  }, [], true );
+
+  it('TINY-9723: Menu toolbar items should not be tabstoppable', async () => {
+    const editor = hook.editor();
+    TinyUiActions.clickOnToolbar(editor, '.tox-tbtn[title="Reveal or hide additional toolbar items"]');
```

Comment:
This line was previously `TinyUiActions.clickOnToolbar(editor, '.tox-tbtn[title="More..."]');`. Had to change to new title